ledger.go

```diff
@@ -86,17 +80,14 @@ func (e EvmosSECP256K1) GetAddressPubKeySECP256K1(hdPath []uint32, hrp string) (
 		return []byte{}, "", errors.New("unable to derive Ledger address, please open the Ethereum app and retry")
 	}
 
-	bech32AddressBytes, err := bech32.ConvertBits(account.Address.Bytes(), 8, 5, true)
+	address, err := sdk.Bech32ifyAddressBytes(hrp, account.Address.Bytes())
 	if err != nil {
-		return []byte{}, "", fmt.Errorf("unable to convert address to 32-bit representation: %w", err)
+		return []byte{}, "", err
```

Comment:
Nit: we could return `nil` instead of an empty slice
```suggestion
		return nil, "", err
```